sqlglot/lineage.py

```diff
@@ -159,6 +159,11 @@ def to_node(
         if upstream:
             upstream.downstream.append(node)
 
+        # if the select is a star add all scope sources as downstreams
+        if select.is_star:
+            for source in scope.sources.values():
+                node.downstream.append(Node(name=select.sql(), source=source, expression=source))
```

Comment:
@tobymao do we care if the `name` is set to dialect-unaware SQL in `Node`s?

Reply:
it’s just going be select * right?

Reply:
Yeah, though you can also have star projections like ``` `t`.* ```